docs/flagd_configuration.md

```diff
@@ -0,0 +1,63 @@
+# Flagd Configuration
+
+The injected flagd sidecar is configured using the `FlagdConfiguration` CRD, the `openfeature.dev/flagdconfiguration` annotation is used to assign `Pods` with their respective `FlagdConfiguration` CRs. The annotation value is a comma separated list of values following one of 2 patterns: {NAME} or {NAMESPACE}/{NAME}. If no namespace is provided, it is assumed that the CR is within the same namespace as the deployed pod, for example:
+```
+    metadata:
+        namespace: test-ns
+        annotations:
+            openfeature.dev/enabled: "true"
+            openfeature.dev/flagdconfiguration:"config-A, test-ns-2/config-B"
+```
+In this example, 2 CRs are being used to configure the injected flagd container, `config-A` (which is assumed to be in the namespace `test-ns`) and `config-B` from the `test-ns-2` namespace, with `config-B` taking precedence in the configuration merge.
+
+## FlagdConfiguration
+
+The flagd configuration CRD contains the following fields:
+
+| Field      | Behavior | Type | 
+| ----------- | ----------- | ----------- |
+| MetricsPort      | Defines the port for flagd to serve metrics on, defaults to 8013       | `int32`       |
+| Port   | Defines the port for flagd to listen on, defaults to 8014        | `int32`        |
+| SocketPath   | Defines the unix socket path to listen on        | `string`       |
+| SyncProviderArgs   | String arguments passed to flagd on startup, documentation can be found [here](https://github.com/open-feature/flagd/blob/main/docs/configuration/configuration.md)        | `array of strings`, key values separated by `=`, e.g `key=value`       |
+| Evaluator   | Sets the flagd evaluator, defaults to `json`        | `string`       |
+| Image   | Allows for the flagd image to be overridden, defaults to `ghcr.io/open-feature/flagd`        | `string`       |
+| Tag   |  Tag to be appended to the flagd image, defaults to `main`        | `string`       |
+
+## Configuration Merging
+
+When multiple `FlagdConfigurations` are passed the configurations are merged, the last `CR` will take precedence over the first, with any configuration from the deprecated `FlagDSpec` field of the `FeatureFlagConfiguration` CRD taking the lowest priority. 
+An example of this behavior can be found below:
+```
+    metadata:
+        annotations:
+            openfeature.dev/enabled: "true"
+            openfeature.dev/flagdconfiguration:"config-A, config-B"
+```
+Config-A:
+```
+apiVersion: core.openfeature.dev/v1alpha2
+kind: FlagdConfiguration
+metadata:
+    name: test-configuration
+spec:
+    metricsPort: 8080
+    tag: latest
+```
+Config-B:
+```
+apiVersion: core.openfeature.dev/v1alpha2
+kind: FlagdConfiguration
+metadata:
+    name: test-configuration
+spec:
+    Port: 8000
```

Comment:
```suggestion
    port: 8000
```

Reply:
This casing causes validation errors for me (same as https://github.com/open-feature/open-feature-operator/pull/276/files#r1048735805)